components/script/textinput.rs

```diff
@@ -850,7 +854,7 @@ impl<T: ClipboardProvider> TextInput<T> {
         } else {
             vec!(content)
         };
-        if update_text_cursor {
+        if let ChangeEditPoint::Change =  *update_text_cursor {
```

Comment:
We should be able to use `if update_text_cursor == ChangeEditPoint::Change {` instead.